### ISSUE TITLE
TEST PHASE

### ISSUE THREAD

**Issue by Ry-C123** [Ry-C123/Freefall#2]
Test HPC runtime for

- different particle numbers (10,50,500,5000,50000)
- different number of CPUs (1,4,10, max available)
- **number of time steps (100, 1000, 10000, ... 1000000)**
- _Do the analysis and keep the results transparent._
- _Make the plots_

---

- **Recreate results from Veras (2014)**
- _Make the plots_

**Comment by Ry-C123** [comment]
TEST 1: FAILED!
^ V0.1.0 is too slow when used in parallel. 

Idea: exploiting atomicity and evolving particles independently and writing to files simultaneously. Corruption can be avoided by making sure simulations do not run on too long. 

Some safeguarding may need to be put in place

**Comment by Ry-C123** [comment]
TEST1:PASS!
V0.3.0 is fast enough and stable. Now have to recreate known results from other papers

**Comment by Ry-C123** [comment]
TEST2 is based on writing stability. 
Duplicate of #7

**Comment by Ry-C123** [comment]
TEST2: FAILED!
V0.4.0 is still getting write errors. Is highly dependent on number of cores and number of particles.

**Comment by Ry-C123** [comment]
Revision TEST1: PASSED

TEST2: PASSED

TEST3: FAILED
spin ustable

**Comment by Ry-C123** [comment]
V0.8.0 - Spin Stabilised 
Successfully recreates Veras 2014 - will appear in paper :D
